.circleci/config.yml

```diff
@@ -173,7 +173,7 @@ jobs:
       - run:
           name: Run editor acceptance tests (eks)
           command: |
-            EDITOR_APP=https://fb-editor-test.apps.live.cloud-platform.service.justice.gov.uk
+            EDITOR_APP=https://testable-metadata-presenter.apps.live.cloud-platform.service.justice.gov.uk
```

Comment:
Do we still want this in?

Reply:
Yeah I think so! chicken-egg as explained on slack 😄